### DISCUSSION TITLE
Changing source URL

### DISCUSSION THREAD

**Discussion by cortosi** [obs-websocket-community-projects/obs-websocket-js#319] [Q&A] [answered]
Hello!

Is it possible to change a source URL? 
For example changing the URL of an image source.. i didn't find anything on the documentation, maybe i missed it (or it is just not possible).

Thank you in advice.

**Comment by Stananas** [comment, marked as answer]
Hello! After a few search, you can change it with:
    await obs.callBatch([
        {
            requestType: "SetInputSettings",
            requestData: {
                inputName: "itemname",
                overlay: true,
                inputSettings: {
                    url: "your url"
                }
            }
        }
    ])
